src/main/resources/assets/forge/loot_tables/grass_seed.json

```diff
@@ -0,0 +1,3 @@
+{
+  "pools": []
```

Comment:
Couldn't we put the current default value in `seedList` here, so forge isn't relying on the deprecated impl?

Reply:
Will see how feasible is that - given that I need to cover the fortune enchantment support as well.